src/app/zap_cluster_list.json

```diff
@@ -366,6 +367,7 @@
         "WATER_HEATER_MANAGEMENT_CLUSTER": ["water-heater-management-server"],
         "WATER_HEATER_MODE_CLUSTER": ["mode-base-server"],
         "WI_FI_NETWORK_DIAGNOSTICS_CLUSTER": [],
+        "WI_FI_NETWORK_MANAGEMENT_CLUSTER": ["wifi-network-management-server"],
```

Comment:
This is the directory for the server code right? So how was this working previously? The changes you made were entirely to the zap file.

Reply:
Actually, it seems that this cluster is repeated, there is a `WIFI_NETWORK_MANAGEMENT_CLUSTER` defined a couple of lines above this one. Maybe that is the reason why this was working before this addition.

Reply:
Well, we should not have these multiple entries....

Reply:
Agree, it seems that this was caused by the way Pascal case is handled in the name of the cluster, I removed the repeated entries for this cluster and just keep the ones that were there before the change.